src/components/Nav/Nav.module.css

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.nav-bar {
+  border-bottom: var(--cpd-border-width-1) solid var(--cpd-color-gray-400);
+  margin: var(--cpd-space-6x) 0;
+  padding: 0;
+}
+
+.nav-bar-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: var(--cpd-space-3x);
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-tab {
```

Comment:
The font on these isn't right, I think it needs a more explicit font declaration:

![Screenshot 2023-09-29 at 09-36-33 UI _ Nav Bar - Basic ⋅ Storybook](https://github.com/vector-im/compound-web/assets/48614497/c3bd5ea8-73f4-41d1-8d4b-934856736352)

Also the underline is unexpected for the `button` items